Unique ids (#883)
radiantearth/stac-spec: item-spec/item-spec.md

| assets     | Map<string, [Asset Object](#asset-object)>                                 | **REQUIRED.** Dictionary of asset objects that can be downloaded, each with a unique key. |
| collection | string                                                                     | The `id` of the STAC Collection this Item references to (see [`collection` relation type](#relation-types)). This field is *required* if such a relation type is present. This field provides an easy way for a user to search for any Items that belong in a specified Collection. |

### Additional Field Information

#### stac_version

In general, STAC versions can be mixed, but please keep the [recommended best practices](../best-practices.md#mixing-stac-versions) in mind.

> #### id

Reviewer: I'd align with the order in the table and move this below the stac_extensions.

Reviewer: @cholmes I guess you've not seen this?

> It is important that an Item identifier is unique within a collection, and that the 
> [Collection identifier](../collection-spec/collection-spec.md#id) in turn is unique globally. Then the two can be combined to
> give a globally unique identifier. Items are *[strongly recommended](#collections)* to have collections, and not having one makes
> it more difficult to be used in the wider STAC ecosystem. If an Item does not have a collection then it should strive to provide
> a globally unique identifier, with a long, unique prefix, to have a better chance of being globally unique.

Reviewer 2: ```suggestion
It is important that an Item identifier is unique within a collection, and that the 
[Collection identifier](../collection-spec/collection-spec.md#id) in turn is unique globally. Then the two can be combined to
give a globally unique identifier. Items are *[strongly recommended](#collections)* to have collections, and not having one makes
it more difficult to be used in the wider STAC ecosystem. Iff an Item does not have a Collection, then the Item identifier should be unique within it's root Catalog.
```
